### DISCUSSION TITLE
Questions that 'traditional' peer-reviewers might ask?

### DISCUSSION THREAD

**Discussion by j0sien** [asreview/asreview#1221] [Q&A]
Dear community,

I wish to conduct and publish my systematic literature review using ASreview.
Now my supervisors a bit afraid, and are asking me if any of you have experience with getting your SLR using ASreview published, and what kind of questions peer-reviewers have asked you on the use of ASreview and justification for it.
What were your responses?

Thanks for sharing,

Josien

**Comment by Rensvandeschoot** [comment]
There is a [curated list](https://www.zotero.org/groups/4597652/asreview_public/collections/IWU8ATMK/tags/systematic_review/collection) (maintained by @boer0107) of papers being published using ASReview. Also, some tips and tricks are discussed in a [discussion thread](https://github.com/asreview/asreview/discussions/979), including the recommendations by PRISMA about what to report when using software like ASReview. 

If reviewers ask questions, feel free to post these on the discussion platform, and the community can help you to answer their questions!
